game.py

```diff
@@ -0,0 +1,126 @@
+        resignation = input()
+        if resignation == "y":
+          print(current_player + " has resigned. " + opposing_player + " wins!")
+          time.sleep(3)
+          return "break"
+        if resignation == "n":
+          pass
+    elif user_input == "draw":
+      pass
+    elif user_input == "save":
+      pass
+    elif user_input == "undo":
+      pass
+    elif user_input == "board":
+      board.ascii.draw_board(board)
+    elif user_input.startswith("move ") and len(user_input) == 13 and user_input[5:7] in board.algebraic_squares and user_input[11:13] in board.algebraic_squares:
+      from_square = user_input[5:7]
+      to_square = user_input[11:13]
+      from_square_number = convert_to_square_number(from_square)
+      to_square_number = convert_to_square_number(to_square)
+      print(board.squares[from_square_number].piece)
+      print(board.squares[to_square_number].piece)
+      print(board.squares[from_square_number].piece_moves)
+      time.sleep(3)
+      if board.squares[from_square_number].piece.is_white == board.white_to_move:
+        board.squares[from_square_number].get_piece_moves(board)
+        print("Same color piece")
+        time.sleep(3)
+        if to_square_number in board.squares[from_square_number].piece_moves:
+          move_pieces(board, from_square_number, to_square_number)
+          print("Is white to move?")
+          print(board.white_to_move)
+          time.sleep(3)
+          break
+        else:
+          print("Invalid move! Please choose another.")
+          time.sleep(3)
+      else:
+        print("It is " + current_player + "'s turn to move!")
+        time.sleep(3)
+    else:
+      print("Invalid entry! Try another command. Enter ? for a list of commands.")
+      time.sleep(3)
+
+def move_pieces(board, from_square_number, to_square_number):
+  board.squares[to_square_number].piece = board.squares[from_square_number].piece
+  board.squares[from_square_number].piece = None
+  board.squares[to_square_number].piece.moved_yet == True
+
+def convert_to_square_number(square):
+  columns = ['a', 'b', 'c', 'd', 'e', 'f', 'g', 'h']
+  rows = ['8', '7', '6', '5', '4', '3', '2', '1']
+  columns_index = columns.index(square[0])
+  rows_index = rows.index(square[1])
+  square_number = rows_index + (8 * columns_index)
+  return square_number
+
+def new_game():
+  board = Board()
+  board.new_game()
+  last_move = None
+  while True:
+    last_move = turn(board, last_move)
+    board.white_to_move = not board.white_to_move
+    if last_move == "break":
+      return
+
+def title_screen():
+   while True:
+    clear()
+    print("Terminal Chess!\n")
+    print("1. Start a New Game\n")
+    print("2. Continue Your Last Game\n")
+    print("3. Exit\n\n")
+    selection = input("Enter your selection: ")
+    match selection:
+        case "1":
+          new_game()
+
+        case "2":
+          pass    
+
+        case "3":
+          break
+
+title_screen()
```

Comment:
Currently, this is a set of methods that pass around a board as well as any other state. Would it be easier if this were a class, so that it could store the current board and other state, rather than having to pass them around?